augur/tasks/data_analysis/insight_worker/setup.py

```diff
@@ -31,7 +31,7 @@ def read(filename):
         'click==8.0.3',
         'scipy>=1.7.3',
         'sklearn==0.0',
-        'numpy>=1.21.0', 
+        'numpy>=1.22.0', 
     ],
```

Comment:
Any specific reason why this machine learning worker uses `numpy>=1.22.0` when the other machine learning workers use `numpy==1.22.0`. Seems like if a new version of numpy is released this could break the insight worker since it would install the newest version

Reply:
No. @ABrain7710 : That is a ghost limb from how it was expressed before.